File: src/main/java/com/p2p/lending/service/BorrowcordService.java
package com.p2p.lending.service;

import java.util.List;

import com.p2p.lending.entity.Borrowcord;

public interface BorrowcordService {
	public List<Borrowcord> selborr(Integer wid);

	public void updborr(Integer wid);

	public void borradd(String yss, Integer beyid, String fangshi);
}

File: src/main/java/com/p2p/lending/service/BorrowcordService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Borrowcord
import org.springframework.stereotype.Service

@Service
interface BorrowcordService {
    fun selborr(wid: Int?): List<Borrowcord>

    fun updborr(wid: Int?)

    fun borradd(yss: String, beyid: Int?, fangshi: String)
}
